snippets/nordic-bt-rpc/README.rst

```diff
@@ -1,11 +1,23 @@
-.. _nordic-rpc-host:
+.. _nordic-bt-rpc:
 
-Nordic RPC Host snippet (nordic-rpc-host)
-#########################################
+Nordic Bluetooth Low Energy Remote Procedure Call snippet (nordic-bt-rpc)
+#########################################################################
 
 Overview
 ********
 
-This snippet allows users to build Bluetooth applications with Bluetooth stack running on Radio
-core. The communication between an application and Bluetooth stack is handled over the Bluetooth
-RPC subsystem.
+This snippet allows users to build Bluetooth applications with Bluetooth stack running on a core dedicated for radio communication.
+For example in the nRF54H20 SOC it is radio core.
+The communication between an application and the Bluetooth stack is handled by the Bluetooth RPC subsystem.
+
+Supported SoCs
+**************
+
+Currently the following SoCs from Nordic Semiconductor are supported for use with the snippet:
+
+* :ref:`nRF5340 <programming_board_names>`
+* :ref:`nRF54H20 <programming_board_names>`
```

Comment:
Alternatively, you can link directly to these SoCs as follows:

* :ref:`zephyr:nrf5340dk_nrf5340`
* :ref:`zephyr:nrf54h20dk_nrf54h20`